src/microbe_stage/components/MicrobeControl.cs

```diff
@@ -394,8 +394,11 @@ public static void SetMucocystState(this ref MicrobeControl control,
             // TODO: is it a good idea to allocate a delegate here?
             colony.PerformForOtherColonyMembersThanLeader(m =>
                 m.Get<MicrobeControl>()
-                    .SetMucocystState(ref m.Get<OrganelleContainer>(), ref m.Get<CompoundStorage>(), m, state,
+                    .SetMucocystState(ref m.Get<OrganelleContainer>(), ref m.Get<CompoundStorage>(), in m, state,
                         mucilageCompound));
+
+            if (entity == colony.Leader)
+                SetStateColonyAware(ref control, entity, state ? MicrobeState.MucocystShield : MicrobeState.Normal);
```

Comment:
Upon testing, I don't think this change actually makes sense because this overrides mucocyst amount checks in the colony members...